apps/test/integration/util/levelTestTypes.js

```diff
@@ -0,0 +1,22 @@
+import tickWrapper from './tickWrapper';
+import {TestResults} from '@cdo/apps/constants';
+
+export function testApplabConsoleOutput({testName, source, expect, ticks=2}) {
```

Comment:
worth adding param tags for these?

Reply:
Yeah, probably.  I figured it might be nice to build up a collection of level test generators, for common cases like this.  Documenting the first one well is probably an important step. :laughing:

Reply:
nice abstraction!